front-end/src/components/UI/Hamburger/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cl from './NavBar.module.css'
+import NavButton from './NavButton/NavButton';
+
+const NavBar = ({visible, callback}) => {
```

Comment:
We send this callback through three components, could we use state management library to simplify this?

Reply:
No, maybe in next projects